chainerx_cc/chainerx/testing/array.h

```diff
@@ -57,7 +57,7 @@ class ArrayBuilder {
             Strides strides = builder.GetStrides<T>();
             int64_t total_size = shape.GetTotalSize();
             size_t n_bytes = internal::GetRequiredBytes(shape, strides, sizeof(T));
-            auto ptr = std::make_unique<uint8_t[]>(n_bytes);
+            auto ptr = std::shared_ptr<uint8_t>(new uint8_t[n_bytes], std::default_delete<uint8_t[]>());
```

Comment:
This one as well. Sorry I forgot to send it.
```suggestion
            std::shared_ptr<uint8_t> ptr{new uint8_t[n_bytes], std::default_delete<uint8_t[]>()};
```